pages/core-concepts/introduction-to-cypress.md

```diff
@@ -0,0 +1,603 @@
+# Introdução ao Cypress
+
+## Cadeias de Comandos
+
+É muito importante entender o mecanismo que o Cypress usa para encadear comandos.
+Ele gerencia uma cadeia de Promises para você, e cada comando gera um "sujeito"
+para o próximo comando, até que a cadeia termine ou seja encontrado um erro.
+O desenvolvedor geralmente não precisa usar Promises diretamente, mas é útil entender como elas funcionam!
+
+### Interagindo com elementos
+
```

Comment:
@edmundobiglia você precisa adicionar TODO's acima de cada bloco de texto que tem link, pois futuramente quando essa página for traduzida a gente vai saber onde teremos que atualizar. Entende?
Exemplo:
`[//]: <> (TODO - Adicionar link - integração github)`